addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1297,8 +1301,15 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
                 }
                 return (['text', 'email', 'tel', 'url', 'search', 'password', 'number'].includes(dependencyEl.type)
                     || dependencyEl.nodeName === 'TEXTAREA') && !['set', '!set'].includes(this.$target[0].dataset.visibilityComparator);
-            case 'hidden_condition_no_text_opt':
-                return dependencyEl && (dependencyEl.type === 'checkbox' || dependencyEl.type === 'radio' || dependencyEl.nodeName === 'SELECT');
+            case 'hidden_condition_no_text_opt': {
+                const containerEl = dependencyEl?.closest(".s_website_form_field");
+                const fieldType = containerEl?.dataset.type;
+                return dependencyEl && (
+                    ["checkbox", "radio"].includes(dependencyEl.type) ||
+                    dependencyEl.nodeName === 'SELECT' ||
+                    fieldType === "record"
+                );
+            }
```

Comment:
My linter suggests me these changes. Could you check if your linter is well configured? That's nipticking but it's always like that the first times :)
```suggestion
            case "hidden_condition_no_text_opt": {
                const containerEl = dependencyEl?.closest(".s_website_form_field");
                const fieldType = containerEl?.dataset.type;
                return (
                    (dependencyEl && ["checkbox", "radio"].includes(dependencyEl.type)) ||
                    dependencyEl.nodeName === "SELECT" ||
                    fieldType === "record"
                );
            }
```

Reply:
Hey, thanks for your feedback. I fixed all linting errors and properly set up my linter.